accounts/views/profile.py

```diff
@@ -0,0 +1,62 @@
+from django.http import Http404
+from django.contrib.auth.decorators import login_required
+from django.urls import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.views.generic import UpdateView
+
+from accounts.forms import EmployeeEditProfile, EmployerEditProfile
+
+from accounts.models import EmployeeProfile, EmployerProfile
+
+
+class UpdateEmployeeProfileView(UpdateView):
+    """
+    Update employee/jobseeker profile info
+    """
+    model = EmployeeProfile
+    form_class = EmployeeEditProfile
+    context_object_name = 'employee'
+    template_name = 'accounts/profile/employee/../../templates/accounts/profile/edit-profile.html'
```

Comment:
if a template path is specified in settings, then we can just use the relative paths.

Reply:
this piece of code was mistakenly pushed.
Correct path in the project is :
template_name = 'accounts/profile/edit_profile.html'